src/nodes/victron-virtual.js

```diff
@@ -804,231 +802,229 @@ module.exports = function (RED) {
+              const switchOutputPropertyKey = `SwitchableOutput/output_1/${name}`
+              ifaceDesc.properties[switchOutputPropertyKey] = { type, format, persist }
 
-                let propValue = value
-                if (name === 'Name') {
-                  // Find the format function for Settings/Type
-                  const typeProp = baseProperties.find(p => p.name === 'Settings/Type')
-                  let typeLabel = `Switch ${i}`
-                  if (typeProp && typeof typeProp.format === 'function') {
-                    typeLabel = typeProp.format(switchType)
-                  }
-                  propValue = typeLabel
+              let propValue = value
+              if (name === 'Name') {
+                // Find the format function for Settings/Type
+                const typeProp = baseProperties.find(p => p.name === 'Settings/Type')
+                let typeLabel = 'Switch'
+                if (typeProp && typeof typeProp.format === 'function') {
+                  typeLabel = typeProp.format(switchType)
                 }
-                if (name === 'Settings/Type') propValue = switchType
+                propValue = typeLabel
+              }
+              if (name === 'Settings/Type') propValue = switchType
 
-                iface[switchOutputPropertyKey] = propValue !== undefined ? propValue : 0
+              iface[switchOutputPropertyKey] = propValue !== undefined ? propValue : 0
 
-                if (name === 'Settings/ValidTypes') {
-                  // Only allow the currently selected switch type in the GUI.
-                  // This sets /ValidTypes to a bitmask with only the current type allowed.
-                  // See: https://github.com/victronenergy/dbus-victron-virtual/issues/XXX
-                  // Example: If switchType is 2, then 1 << 2 = 4, so only type 2 is valid.
-                  iface[switchOutputPropertyKey] = 1 << switchType
-                }
-              })
+              if (name === 'Settings/ValidTypes') {
+                // Only allow the currently selected switch type in the GUI.
+                // This sets /ValidTypes to a bitmask with only the current type allowed.
+                // See: https://github.com/victronenergy/dbus-victron-virtual/issues/XXX
+                // Example: If switchType is 2, then 1 << 2 = 4, so only type 2 is valid.
+                iface[switchOutputPropertyKey] = 1 << switchType
+              }
+            })
 
-              // Set CustomName and Group from config
-              const customNameKey = `SwitchableOutput/output_${i}/Settings/CustomName`
-              ifaceDesc.properties[customNameKey] = {
-                type: 's',
+            // Set CustomName and Group from config
+            const customNameKey = 'SwitchableOutput/output_1/Settings/CustomName'
+            ifaceDesc.properties[customNameKey] = {
+              type: 's',
+              persist: true
```

Comment:
As discussed: It appears that `persist: true` practically makes the config property un-editable. On the UI, it seems you can update, but it has no effect on the property; the issue can be confirmed via `dbus-spy`.

See [this PR](https://github.com/victronenergy/node-red-contrib-victron/pull/306) to fix this.